scripts/gitlab-mirror/mirror

```diff
@@ -22,6 +22,6 @@ git push --mirror $MIRROR
 # Update the mirror
 while true
 do
-git remote update
+git remote update --prune
```

Comment:
Oooo

Reply:
Should be safe :) If we've deleted a branch on Github, it shouldn't be a problem if the pipeline on Gitlab gets deleted too